src/directives/formly-form.test.js

```diff
@@ -280,6 +280,48 @@ describe('formly-form', () => {
       })
     })
 
+    it(`should be possible to use a wrapper & templateOptions in a fieldGroup`, () => {
+      formlyConfig.setWrapper({
+        name: 'panel',
+        template: `<div class="panel">
+            <div class="heading">
+              Panel Title: {{options.templateOptions.title}}
```

Comment:
http://jsbin.com/kutucisiqo/1/edit?html,js,output

The `$scope.to` variable seems to be only set in the `simplifyLife()` (see https://github.com/formly-js/angular-formly/blob/master/src/directives/formly-field.js#L95) variable, which is only set if the field is not a fieldGroup (https://github.com/formly-js/angular-formly/blob/master/src/directives/formly-field.js#L45)